source/components/multiStepIndicator/multiStepIndicator.html

```diff
@@ -1,11 +1,15 @@
-<div class="multi-step" ng-class="{ 'numbered': breadcrumb.numbered, 'checked': breadcrumb.checked }">
+<div class="multi-step"
+	[class.numbered]="numbered"
+	[class.checked]="checked">
 	<ol>
-		<li ng-repeat="step in breadcrumb.steps" ng-click="breadcrumb.onClick(step)"
-			ng-class="{ 'completed': step.getCompleted(), 'current': step.isCurrent, 'active': !step.inactive && !breadcrumb.anyLoading() }">
+		<li *ngFor="let step of steps" (click)="onClick(step)"
+			[class.completed]="step.getCompleted()"
+			[class.current]="step.isCurrent"
+			[class.active]="step.isActive && !anyLoading()">
 			<div class="wrap">
-				<p class="badge" ng-show="step.hasCount">{{step.count()}}</p>
-				<p class="error" ng-if="!step.getValid()"></p>
-				<p class="title">{{step.title}} <rl-busy loading="step.loading"></rl-busy></p>
+				<p class="badge" *ngIf="step.count">{{step.count()}}</p>
+				<p class="error" *ngIf="!step.getValid()"></p>
+				<p class="title">{{step.title}} <rl-busy [loading]="step.isLoading"></rl-busy></p>
```

Comment:
<rl-busy> is not a native element, so you'll have to import and use our busy component in angular 2.

```
import { BusyComponent } from '../busy/busy';

// your component definition
@Component({
    directives: [BusyComponent],
})

// template
<rlBusy></rlBusy>
or
<rlBusy [loading]="true"></rlBusy>
```

Because Angular 2 handles component interactions well, we also decided to add a `trigger` API to the busy component. So instead of managing a loading flag at all, you could just pass the click promise to the `trigger` method and it'll spin until the promise completes. Although, this might not work very well since each step has its own busy.

```
@ViewChild(BusyComponent) busy: BusyComponent;

// in an action method
let myPromse = doSomething();
this.busy.trigger(myPromise); // spins until promise resolves
```

This could be done with a child component for the individual steps, but that would take more work. Otherwise it's fine as-is.